notebooks/Data_Attribute_Recommendation_Regression_Model_Template.ipynb

```diff
@@ -0,0 +1,1933 @@
+    }
+   },
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please log in to your trial account: https://cockpit.eu10.hana.ondemand.com/trial/\n",
```

Comment:
We may have to update this URL to US10. I am currently creating my trial account from scratch so I will check later.

Reply:
Done. Followed your regression notebook to get this in line.